src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/Microsoft.Extensions.Logging.Console.Tests.csproj

```diff
@@ -5,10 +5,11 @@
     <TestRuntime>true</TestRuntime>
     <EnableDefaultItems>true</EnableDefaultItems>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <_WasmPThreadPoolSize Condition="'$(MonoWasmBuildVariant)' == 'multithread'">100</_WasmPThreadPoolSize>
```

Comment:
😱

Reply:
the test is allocating lot of threads. I suspect that we are still not freeing them fast enough.
If we adopt deputy design, we could have dynamic thread allocation.
It's obvious this is not sustainable. On the other hand, this is just a test, not production code.